creator-node/src/fileManager.js

```diff
@@ -287,7 +286,6 @@ async function saveFileForMultihashToFS (serviceRegistry, logger, multihash, exp
     }
 
     _printDecisionTreeObj(decisionTree, logger)
-    return expectedStoragePath
```

Comment:
this wasn't used anywhere, plus it is passed in as a param so returning it is useless